cli/cli/kurtosis_config/kurtosis_config_provider.go

```diff
@@ -35,6 +36,7 @@ func (configProvider *KurtosisConfigProvider) GetOrInitializeConfig() (*resolved
 		if err != nil {
 			return nil, stacktrace.Propagate(err, "An error occurred executing init interactive config")
 		}
+		email_collector.AskUserForEmailAndLogIt()
```

Comment:
I don't have much context but are we going to skip the prompt from current users? is it the idea?

Reply:
Yes!